Sources/SemanticAnalyzer/SemanticError.swift

```diff
@@ -478,8 +478,23 @@ extension Diagnostic {
 
   static func publicAndVisible(_ variableDeclaration: VariableDeclaration) -> Diagnostic {
     let identName = variableDeclaration.identifier.name
-   return Diagnostic(severity: .error, sourceLocation: variableDeclaration.sourceLocation,
-                     message: "Cannot declare variable '\(identName)' both public and visible")
+    return Diagnostic(severity: .error, sourceLocation: variableDeclaration.sourceLocation,
+                      message: "Cannot declare variable '\(identName)' both public and visible")
+  }
+
+  static func invalidExternalCallHyperParameter(_ identifier: Identifier) -> Diagnostic {
```

Comment:
Call this `invalidExternalCallConfigurationParameter` + the others or rename occurrences of `ConfigurationParameter` in the codebase for consistency

Reply:
Changed `configurationParameter` to `hyperParameter` throughout codebase.

Reply:
anise